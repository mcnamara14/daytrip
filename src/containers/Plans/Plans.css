.plansContainer {
  width: 86%;
  margin: 0 auto;
  padding-top: 60px;
  display: flex;
  flex-direction: column;
}

.restaurantImage {
  width: 90%;
  height: 190px;
  border-radius: 5px 5px 0 0;
  display: block;
  border-radius: 5px;
  background-size: cover;
  background-position: center;
  border: 1px solid #eee;
}

.plansContainer h3 {
  font-size: 1.5rem;
  color: #494949;
  text-transform: uppercase;
  text-align: left;
}

.plansContainer h3 {
color: #7c7c7c;
font-size: 1.2rem;
text-transform: uppercase;
}