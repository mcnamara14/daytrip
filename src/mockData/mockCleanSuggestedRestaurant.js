export const mockCleanSuggestedRestaurant = [
  {
    address: "1543 Champa St Denver, CO 80202",
    image_url: "https://s3-media2.fl.yelpcdn.com/bphoto/Twaes8jv0U3XOgGu6H9d-A/o.jpg",
    name: "Union Lodge No.1",
    price: "$$",
    rating: 5,
    review_count: 382
  }
];

export const mockDirtySuggestedRestaurant = [
  {
    image_url: "https://s3-media2.fl.yelpcdn.com/bphoto/Twaes8jv0U3XOgGu6H9d-A/o.jpg",
    name: "Union Lodge No.1",
    price: "$$",
    rating: 5,
    review_count: 382,
    location: {
      display_address : [
        "1543 Champa St",
        "Denver, CO 80202"
      ]
    }
  }
];