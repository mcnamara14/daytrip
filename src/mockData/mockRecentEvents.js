export const mockDirtyRecentEvents = {
  events: [
    {
      title: "T Swift",
      image: "https://s1.ticketm.allswifty.jpg",
      price: "$300",
      venue: "Boulder Theater",
      date: "2018-05-30 8:00 PM"
    }
  ]
};

