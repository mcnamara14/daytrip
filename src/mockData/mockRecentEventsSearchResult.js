export const mockCleanRecentEventsSearchResult = {
  "options": [
    {"id": "Zrt9dk32l", 
      "label": "T SwiftBoulder Theater2018-05-30 2018-05-27T17:13:38-06:00", 
      "value": 0}
  ]
}

export const mockCleanRecentEventsSearchResultNoStart = {
  "options": [
    {"id": "Zrt9dk32l", 
      "label": "T SwiftBoulder TheaterTBA", 
      "value": 0}
  ]
}
