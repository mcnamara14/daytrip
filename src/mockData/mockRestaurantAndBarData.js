export const mockCleanRestaurantAndBar = [
  { address: "1420 E 18th Ave Denver, CO 80218",
    image_url: "https://s3-media3.fl.yelpcdn.com/tg/o.jpg",
    name: "Beebs",
    price: "$$",
    rating: 4.5,
    review_count: 450
  }
]