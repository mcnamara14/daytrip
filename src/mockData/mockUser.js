export const mockUser = {
  userId: "tvcmzDN3SoZ6vU07a1w9vY3S8zt1",
  email: "mcnamara14@gmail.com",
  city: "San Diego",
  state: "CA"
};