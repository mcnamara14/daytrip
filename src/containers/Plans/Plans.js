import React, { Component } from 'react';
import { connect } from 'react-redux';
import './Plans.css';
import Stop from '../Stop/Stop';
import PropTypes from 'prop-types';
import * as firebase from 'firebase';
import 'firebase/database';
import { withRouter } from 'react-router-dom';

export class Plans extends Component {
  selectPlan = (
    restaurantId,
    restaurantName, 
    restaurantRating, 
    restaurantAddress, 
    restaurantPrice, 
    restaurantReviewCount, 
    restaurantImage,
    barId,
    barName, 
    barRating, 
    barAddress, 
    barPrice, 
    barReviewCount, 
    barImage,
    id,
    title, 
    image, 
    venue, 
    date,
    eventPrice,
    planNum,
    url
  ) => {
    let firebaseLocation;
    const firebaseRef = firebase.database().ref();

    this.props.user.userId !== null ? firebaseLocation = firebaseRef.child('users').child(this.props.user.userId).child('selectedPlan') :
      firebaseLocation = firebaseRef.child('users').child('anonymous').child('selectedPlan');

    firebaseLocation.update(
      {
        restaurant: {
          id: restaurantId,
          title: restaurantName,
          rating: restaurantRating,
          location: restaurantAddress, 
          price: restaurantPrice, 
          reviews: restaurantReviewCount, 
          image: restaurantImage, 
          type: 'restaurant', 
          index: planNum
        },
        bar: {
          id: barId,
          title: barName,
          rating: barRating,
          location: barAddress, 
          price: barPrice,
          reviews: barReviewCount,
          image: barImage,
          type: 'bar' ,
          index: planNum
        },
        event: {
          id: id, 
          title: title, 
          rating: 0, 
          location: venue, 
          price: eventPrice, 
          reviews: date, 
          image: image, 
          type: 'event', 
          index: planNum,
          tickets: url
        }
      }
    );

    this.props.history.push('/plan');
  }

  getPlans = () => {
    const { 
      suggestedBars, 
      suggestedRestaurants, 
      selectedEvent 
    } = this.props;
    let plans = [];

    if (suggestedBars.length && suggestedRestaurants.length) {
      for (var i = 0; i < this.props.suggestedBars.length; i++) {
        const { 
          id: restaurantId,
          name: restaurantName, 
          rating: restaurantRating, 
          address: restaurantAddress, 
          price: restaurantPrice, 
          review_count: restaurantReviewCount, 
          image_url: restaurantImage 
        } = this.props.suggestedRestaurants[i];
        const { 
          id: barId,
          name: barName, 
          rating: barRating, 
          address: barAddress, 
          price: barPrice, 
          review_count: barReviewCount, 
          image_url: barImage 
        } = this.props.suggestedBars[i];
        const { 
          id,
          title, 
          image, 
          venue, 
          date,
          url } = this.props.selectedEvent;
        const eventPrice = selectedEvent.price;
        const planNum = i + 1;

        plans.push(<div key={i} className="planContainer">
          <h4>Plan {i + 1}</h4>
          <section className="plan">
            <Stop 
              id={restaurantId}
              title={restaurantName} 
              rating={restaurantRating} 
              location={restaurantAddress} 
              price={restaurantPrice} 
              reviews={restaurantReviewCount} 
              image={restaurantImage} 
              type={'restaurant'} 
              index={planNum}
            />
            <img src={require('./assets/route-img.jpg')} className="routeImg" />
            <Stop
              id={id} 
              title={title} 
              rating={0} 
              location={venue} 
              price={eventPrice} 
              reviews={date} 
              image={image} 
              type={'event'} 
              index={planNum}
            />
            <img src={require('./assets/route-img.jpg')} className="routeImg" />
            <Stop 
              id={barId}
              title={barName} 
              rating={barRating} 
              location={barAddress} 
              price={barPrice} 
              reviews={barReviewCount} 
              image={barImage} 
              type={'bar'} 
              index={planNum}
            />
          </section>
          <button onClick={() => this.selectPlan(
            restaurantId,
            restaurantName, 
            restaurantRating, 
            restaurantAddress, 
            restaurantPrice, 
            restaurantReviewCount, 
            restaurantImage,
            barId,
            barName, 
            barRating, 
            barAddress, 
            barPrice, 
            barReviewCount, 
            barImage,
            id,
            title, 
            image, 
            venue, 
            date,
            eventPrice,
            planNum,
            url
          )}>Select Plan</button>
        </div>);
      }
    }

    return plans;
  }

  render () {
    const { suggestedBars, suggestedRestaurants } = this.props;

    return (
      <section className="suggestedPlans">
        <div className="plansContainer">
          <h3>Suggested Plan</h3>
          { suggestedBars.length && suggestedRestaurants.length ? 
            this.getPlans() : null }
        </div>
      </section>
    );
  }
}

Plans.propTypes = {
  suggestedRestaurants: PropTypes.array,
  suggestedBars: PropTypes.array,
  selectedEvent: PropTypes.object
};

Stop.propTypes = {
  rating: PropTypes.number,
  selectedEvent: PropTypes.object
};

export const mapStateToProps = (state) => ({
  suggestedRestaurants: state.suggestedRestaurants,
  suggestedBars: state.suggestedBars,
  selectedEvent: state.selectedEvent,
  user: state.user
});

export default withRouter(connect(mapStateToProps)(Plans));