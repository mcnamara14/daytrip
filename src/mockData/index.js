import { mockDirtyRecentEvents } from './mockRecentEvents';
import { mockCleanRestaurantAndBarData } from './mockRestaurantAndBarData';

export {
  mockDirtyRecentEvents,
  mockCleanRestaurantAndBarData
}