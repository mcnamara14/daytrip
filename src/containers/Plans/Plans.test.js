import React from 'react';
import ReactDOM from 'react-dom';
import { Plans } from './Plans';
import { shallow } from 'enzyme';
import { mockCleanRestaurantAndBar, mockDirtyRecentEvents } from '../../mockData';

it('should match the snapshot', () => {
  const mockSuggestedBarAndRestaurantData = mockCleanRestaurantAndBar;
  const mockSelectedEvent = mockDirtyRecentEvents.events;

  const wrapper = shallow(<Plans 
    suggestedBars={mockSuggestedBarAndRestaurantData}
    suggestedRestaurants={mockSuggestedBarAndRestaurantData} 
    selectedEvent={mockSelectedEvent}
    />);

  expect(wrapper).toMatchSnapshot();
});
