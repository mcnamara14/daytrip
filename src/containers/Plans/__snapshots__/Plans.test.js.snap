// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plans should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Plans
    selectedEvent={
        Object {
            "_embedded": Object {
              "venues": Array [
                Object {
                  "location": Object {
                    "latitude": "39.735001",
                    "longitude": "-105.020401",
                  },
                  "name": "Boulder Theater",
                },
              ],
            },
            "classifications": Array [
              Object {
                "test": "...",
              },
            ],
            "dates": Object {
              "start": Object {
                "localDate": "2018-05-30",
                "localTime": "08:00:00",
              },
            },
            "id": "Zrt9dk32l",
            "images": Array [
              Object {
                "url": "https://s1.ticketm.allswifty.jpg",
              },
            ],
            "name": "T Swift",
            "priceRanges": Array [
              Object {
                "currency": "USD",
                "max": 12,
                "min": 300,
              },
            ],
            "sales": Object {
              "public": Object {
                "test": "...",
              },
            },
            "type": "event",
            "url": "http://www.ticketsnow.com/PID=2107079",
          }
    }
    suggestedBars={
        Array [
            Object {
              "address": "1420 E 18th Ave Denver, CO 80218",
              "image_url": "https://s3-media3.fl.yelpcdn.com/tg/o.jpg",
              "name": "Beebs",
              "price": "$$",
              "rating": 4.5,
              "review_count": 450,
            },
          ]
    }
    suggestedRestaurants={
        Array [
            Object {
              "address": "1420 E 18th Ave Denver, CO 80218",
              "image_url": "https://s3-media3.fl.yelpcdn.com/tg/o.jpg",
              "name": "Beebs",
              "price": "$$",
              "rating": 4.5,
              "review_count": 450,
            },
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="plansContainer"
>
        <h3>
                Suggested Plan
        </h3>
        <div
                className="planContainer"
        >
                <h4>
                        Plan 
                        1
                </h4>
                <section
                        className="plan"
                >
                        <Stop
                                id={undefined}
                                image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                index={1}
                                location="1420 E 18th Ave Denver, CO 80218"
                                price="$$"
                                rating={4.5}
                                reviews={450}
                                title="Beebs"
                                type="restaurant"
                        />
                        <img
                                alt="Arrow"
                                className="routeImg"
                                src="route-img.jpg"
                        />
                        <Stop
                                id="Zrt9dk32l"
                                image={undefined}
                                index={1}
                                location={undefined}
                                price={undefined}
                                rating={0}
                                reviews={undefined}
                                title={undefined}
                                type="event"
                        />
                        <img
                                alt="Arrow"
                                className="routeImg"
                                src="route-img.jpg"
                        />
                        <Stop
                                id={undefined}
                                image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                index={1}
                                location="1420 E 18th Ave Denver, CO 80218"
                                price="$$"
                                rating={4.5}
                                reviews={450}
                                title="Beebs"
                                type="bar"
                        />
                </section>
                <button
                        onClick={[Function]}
                >
                        Select Plan
                </button>
        </div>
</div>,
      "className": "suggestedPlans",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Suggested Plan
</h3>,
          Array [
            <div
              className="planContainer"
>
              <h4>
                            Plan 
                            1
              </h4>
              <section
                            className="plan"
              >
                            <Stop
                                          id={undefined}
                                          image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                          index={1}
                                          location="1420 E 18th Ave Denver, CO 80218"
                                          price="$$"
                                          rating={4.5}
                                          reviews={450}
                                          title="Beebs"
                                          type="restaurant"
                            />
                            <img
                                          alt="Arrow"
                                          className="routeImg"
                                          src="route-img.jpg"
                            />
                            <Stop
                                          id="Zrt9dk32l"
                                          image={undefined}
                                          index={1}
                                          location={undefined}
                                          price={undefined}
                                          rating={0}
                                          reviews={undefined}
                                          title={undefined}
                                          type="event"
                            />
                            <img
                                          alt="Arrow"
                                          className="routeImg"
                                          src="route-img.jpg"
                            />
                            <Stop
                                          id={undefined}
                                          image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                          index={1}
                                          location="1420 E 18th Ave Denver, CO 80218"
                                          price="$$"
                                          rating={4.5}
                                          reviews={450}
                                          title="Beebs"
                                          type="bar"
                            />
              </section>
              <button
                            onClick={[Function]}
              >
                            Select Plan
              </button>
</div>,
          ],
        ],
        "className": "plansContainer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Suggested Plan",
          },
          "ref": null,
          "rendered": "Suggested Plan",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h4>
                Plan 
                1
</h4>,
              <section
                className="plan"
>
                <Stop
                                id={undefined}
                                image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                index={1}
                                location="1420 E 18th Ave Denver, CO 80218"
                                price="$$"
                                rating={4.5}
                                reviews={450}
                                title="Beebs"
                                type="restaurant"
                />
                <img
                                alt="Arrow"
                                className="routeImg"
                                src="route-img.jpg"
                />
                <Stop
                                id="Zrt9dk32l"
                                image={undefined}
                                index={1}
                                location={undefined}
                                price={undefined}
                                rating={0}
                                reviews={undefined}
                                title={undefined}
                                type="event"
                />
                <img
                                alt="Arrow"
                                className="routeImg"
                                src="route-img.jpg"
                />
                <Stop
                                id={undefined}
                                image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                index={1}
                                location="1420 E 18th Ave Denver, CO 80218"
                                price="$$"
                                rating={4.5}
                                reviews={450}
                                title="Beebs"
                                type="bar"
                />
</section>,
              <button
                onClick={[Function]}
>
                Select Plan
</button>,
            ],
            "className": "planContainer",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Plan ",
                  1,
                ],
              },
              "ref": null,
              "rendered": Array [
                "Plan ",
                1,
              ],
              "type": "h4",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Stop
                    id={undefined}
                    image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                    index={1}
                    location="1420 E 18th Ave Denver, CO 80218"
                    price="$$"
                    rating={4.5}
                    reviews={450}
                    title="Beebs"
                    type="restaurant"
/>,
                  <img
                    alt="Arrow"
                    className="routeImg"
                    src="route-img.jpg"
/>,
                  <Stop
                    id="Zrt9dk32l"
                    image={undefined}
                    index={1}
                    location={undefined}
                    price={undefined}
                    rating={0}
                    reviews={undefined}
                    title={undefined}
                    type="event"
/>,
                  <img
                    alt="Arrow"
                    className="routeImg"
                    src="route-img.jpg"
/>,
                  <Stop
                    id={undefined}
                    image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                    index={1}
                    location="1420 E 18th Ave Denver, CO 80218"
                    price="$$"
                    rating={4.5}
                    reviews={450}
                    title="Beebs"
                    type="bar"
/>,
                ],
                "className": "plan",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "id": undefined,
                    "image": "https://s3-media3.fl.yelpcdn.com/tg/o.jpg",
                    "index": 1,
                    "location": "1420 E 18th Ave Denver, CO 80218",
                    "price": "$$",
                    "rating": 4.5,
                    "reviews": 450,
                    "title": "Beebs",
                    "type": "restaurant",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Arrow",
                    "className": "routeImg",
                    "src": "route-img.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "id": "Zrt9dk32l",
                    "image": undefined,
                    "index": 1,
                    "location": undefined,
                    "price": undefined,
                    "rating": 0,
                    "reviews": undefined,
                    "title": undefined,
                    "type": "event",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "Arrow",
                    "className": "routeImg",
                    "src": "route-img.jpg",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "id": undefined,
                    "image": "https://s3-media3.fl.yelpcdn.com/tg/o.jpg",
                    "index": 1,
                    "location": "1420 E 18th Ave Denver, CO 80218",
                    "price": "$$",
                    "rating": 4.5,
                    "reviews": 450,
                    "title": "Beebs",
                    "type": "bar",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
              ],
              "type": "section",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Select Plan",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": "Select Plan",
              "type": "button",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="plansContainer"
>
          <h3>
                    Suggested Plan
          </h3>
          <div
                    className="planContainer"
          >
                    <h4>
                              Plan 
                              1
                    </h4>
                    <section
                              className="plan"
                    >
                              <Stop
                                        id={undefined}
                                        image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                        index={1}
                                        location="1420 E 18th Ave Denver, CO 80218"
                                        price="$$"
                                        rating={4.5}
                                        reviews={450}
                                        title="Beebs"
                                        type="restaurant"
                              />
                              <img
                                        alt="Arrow"
                                        className="routeImg"
                                        src="route-img.jpg"
                              />
                              <Stop
                                        id="Zrt9dk32l"
                                        image={undefined}
                                        index={1}
                                        location={undefined}
                                        price={undefined}
                                        rating={0}
                                        reviews={undefined}
                                        title={undefined}
                                        type="event"
                              />
                              <img
                                        alt="Arrow"
                                        className="routeImg"
                                        src="route-img.jpg"
                              />
                              <Stop
                                        id={undefined}
                                        image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                        index={1}
                                        location="1420 E 18th Ave Denver, CO 80218"
                                        price="$$"
                                        rating={4.5}
                                        reviews={450}
                                        title="Beebs"
                                        type="bar"
                              />
                    </section>
                    <button
                              onClick={[Function]}
                    >
                              Select Plan
                    </button>
          </div>
</div>,
        "className": "suggestedPlans",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3>
              Suggested Plan
</h3>,
            Array [
              <div
                className="planContainer"
>
                <h4>
                                Plan 
                                1
                </h4>
                <section
                                className="plan"
                >
                                <Stop
                                                id={undefined}
                                                image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                                index={1}
                                                location="1420 E 18th Ave Denver, CO 80218"
                                                price="$$"
                                                rating={4.5}
                                                reviews={450}
                                                title="Beebs"
                                                type="restaurant"
                                />
                                <img
                                                alt="Arrow"
                                                className="routeImg"
                                                src="route-img.jpg"
                                />
                                <Stop
                                                id="Zrt9dk32l"
                                                image={undefined}
                                                index={1}
                                                location={undefined}
                                                price={undefined}
                                                rating={0}
                                                reviews={undefined}
                                                title={undefined}
                                                type="event"
                                />
                                <img
                                                alt="Arrow"
                                                className="routeImg"
                                                src="route-img.jpg"
                                />
                                <Stop
                                                id={undefined}
                                                image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                                index={1}
                                                location="1420 E 18th Ave Denver, CO 80218"
                                                price="$$"
                                                rating={4.5}
                                                reviews={450}
                                                title="Beebs"
                                                type="bar"
                                />
                </section>
                <button
                                onClick={[Function]}
                >
                                Select Plan
                </button>
</div>,
            ],
          ],
          "className": "plansContainer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Suggested Plan",
            },
            "ref": null,
            "rendered": "Suggested Plan",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": "0",
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h4>
                  Plan 
                  1
</h4>,
                <section
                  className="plan"
>
                  <Stop
                                    id={undefined}
                                    image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                    index={1}
                                    location="1420 E 18th Ave Denver, CO 80218"
                                    price="$$"
                                    rating={4.5}
                                    reviews={450}
                                    title="Beebs"
                                    type="restaurant"
                  />
                  <img
                                    alt="Arrow"
                                    className="routeImg"
                                    src="route-img.jpg"
                  />
                  <Stop
                                    id="Zrt9dk32l"
                                    image={undefined}
                                    index={1}
                                    location={undefined}
                                    price={undefined}
                                    rating={0}
                                    reviews={undefined}
                                    title={undefined}
                                    type="event"
                  />
                  <img
                                    alt="Arrow"
                                    className="routeImg"
                                    src="route-img.jpg"
                  />
                  <Stop
                                    id={undefined}
                                    image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                                    index={1}
                                    location="1420 E 18th Ave Denver, CO 80218"
                                    price="$$"
                                    rating={4.5}
                                    reviews={450}
                                    title="Beebs"
                                    type="bar"
                  />
</section>,
                <button
                  onClick={[Function]}
>
                  Select Plan
</button>,
              ],
              "className": "planContainer",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "Plan ",
                    1,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  "Plan ",
                  1,
                ],
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <Stop
                      id={undefined}
                      image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                      index={1}
                      location="1420 E 18th Ave Denver, CO 80218"
                      price="$$"
                      rating={4.5}
                      reviews={450}
                      title="Beebs"
                      type="restaurant"
/>,
                    <img
                      alt="Arrow"
                      className="routeImg"
                      src="route-img.jpg"
/>,
                    <Stop
                      id="Zrt9dk32l"
                      image={undefined}
                      index={1}
                      location={undefined}
                      price={undefined}
                      rating={0}
                      reviews={undefined}
                      title={undefined}
                      type="event"
/>,
                    <img
                      alt="Arrow"
                      className="routeImg"
                      src="route-img.jpg"
/>,
                    <Stop
                      id={undefined}
                      image="https://s3-media3.fl.yelpcdn.com/tg/o.jpg"
                      index={1}
                      location="1420 E 18th Ave Denver, CO 80218"
                      price="$$"
                      rating={4.5}
                      reviews={450}
                      title="Beebs"
                      type="bar"
/>,
                  ],
                  "className": "plan",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "id": undefined,
                      "image": "https://s3-media3.fl.yelpcdn.com/tg/o.jpg",
                      "index": 1,
                      "location": "1420 E 18th Ave Denver, CO 80218",
                      "price": "$$",
                      "rating": 4.5,
                      "reviews": 450,
                      "title": "Beebs",
                      "type": "restaurant",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Arrow",
                      "className": "routeImg",
                      "src": "route-img.jpg",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "id": "Zrt9dk32l",
                      "image": undefined,
                      "index": 1,
                      "location": undefined,
                      "price": undefined,
                      "rating": 0,
                      "reviews": undefined,
                      "title": undefined,
                      "type": "event",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "Arrow",
                      "className": "routeImg",
                      "src": "route-img.jpg",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "id": undefined,
                      "image": "https://s3-media3.fl.yelpcdn.com/tg/o.jpg",
                      "index": 1,
                      "location": "1420 E 18th Ave Denver, CO 80218",
                      "price": "$$",
                      "rating": 4.5,
                      "reviews": 450,
                      "title": "Beebs",
                      "type": "bar",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                ],
                "type": "section",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Select Plan",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "Select Plan",
                "type": "button",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
